packages/protocol/test/requesters/RrpRequester.sol.js

```diff
@@ -1,27 +1,93 @@
 /* globals context */
 const hre = require('hardhat');
 const { expect } = require('chai');
+const utils = require('../utils');
 
 let roles;
-let rrpRequester;
+let rrpRequester, airnodeRrp;
+let airnodeAddress, airnodeMnemonic, airnodeXpub;
+let sponsorWalletAddress;
 
 beforeEach(async () => {
   const accounts = await hre.ethers.getSigners();
   roles = {
     deployer: accounts[0],
-    mockAirnodeRrp: accounts[1],
```

Comment:
cannot use a EOA anymore because while deploying we are making a call to airnodeRrp to set the sponsor. This also means to test onlyAirnodeRrp we need to make a valid request and also have it fulfilled.